onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1253,7 +1253,14 @@ void addObjectMethods(py::module& m, Environment& env) {
         // TODO: Assumes that the OrtValue is a Tensor, make this generic to handle non-Tensors
         ORT_ENFORCE(ml_value->IsTensor(), "Only OrtValues that are Tensors are currently supported");
 
-        return DataTypeImpl::ToString(ml_value->Get<Tensor>().DataType());
+        // Currently only "tensor" OrtValues are supported
+        std::ostringstream ostr;
+        ostr << "tensor";
+        ostr << "(";
```

Comment:
This was working as is previously because DataTy[peImpl::ToString() palways appended "tensor()" to any primitive type